dap-mode.el

```diff
@@ -1442,13 +1442,22 @@ If ORIGIN is t, return the original configuration without prepopulation"
           (error "There is no debug provider for language %s"
                  (or (plist-get debug-args :type) "'Not specified'"))))))
 
+(defun dap-debug-template-configurations-provider ()
+  dap-debug-template-configurations)
```

Comment:
launch.json params should be on by default.

Reply:
Don't we need to load dap-launch.el first?

Reply:
yes we need unless the function is marked as autoloaded.

Reply:
How do you do that?

Reply:
Fixed in latest master.